rust/lib/Cargo.toml

```diff
@@ -10,6 +10,22 @@ zingoconfig = { workspace = true }
 http = "0.2.4"
 lazy_static = "1.4.0"
 base64 = "0.11.0"
+android_logger = "0.11"
+log = "0.4.8"
+uniffi = { workspace = true, features = ["cli"] }
+
+[build-dependencies]
+uniffi_build = { version = "0.27.0", features = [ "builtin-bindgen" ] }
 
 [dev-dependencies]
 tokio = "1.32.0"
+
+[lib]
+crate-type = ["cdylib", "staticlib"]
+name = "rustlib"
```

Comment:
I wonder if a more appropriate name is "zingo" here. I have noticed a `librustlib.so` which doesnt seem right.
"libzingo" and "uniffi_build::generate_scaffolding("src/zingolib.udl") seems better.

BUT I think this is probably best in a following PR. not a requested change!

Reply:
I agree, would be fantastic if you can create an issue with this, thanks in advance.